storage/bucket.go

```diff
@@ -1362,8 +1362,8 @@ func (b *BucketHandle) LockRetentionPolicy(ctx context.Context) error {
 }
 
 // SetObjectRetention returns a new BucketHandle that allows enabling object
```

Comment:
Maybe rephrase to make it clearer that calling the method means that object retention WILL be enabled on the created bucket (not "allows enabling"). Also, clarify that this has no effect on an already existing bucket.